torch/_inductor/analysis.py

```diff
@@ -0,0 +1,116 @@
+import collections
+from typing import Any, List
+
+import torch
+import torch.fx as fx
+from . import ir
+
+
+def create_fx_from_snodes(snodes: List["scheduler.BaseSchedulerNode"]) -> fx.Graph:
```

Comment:
Is this just a move?  Or was something changed?

Reply:
Just a move with some minor bug fixes. I moved it into here since it changed from being a debug utility to something that's load bearing in optimizations.